src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -81,10 +83,13 @@ Copyright (c) .NET Foundation. All rights reserved.
           PublishDir=$(PublishDir);
           StoreStagingDir=$(StoreStagingDir);
           TargetFramework=$(TargetFramework);
+          RuntimeIdentifier=$(RuntimeIdentifier);
```

Comment:
Note: this change, and the similar one below are fixing a bug to allow the user to set the `RuntimeIdentifier` in their manifest file.  Today it only works if they pass it in on the command line.